share/instruments/instruments.xml

```diff
@@ -10946,6 +10946,29 @@
                   <longName>Double Bass</longName>
                   <shortName>Db.</shortName>
             </Instrument>
+            <Instrument id="octobass">
+                  <longName>Octobass</longName>
+                  <shortName>Ob.</shortName>
```

Comment:
Coincide with oboe?

Reply:
Shortnames to that on occasions

Reply:
Maybe `Otb.`? I don't really like the duplicate

Reply:
OK